src/main/java/org/team1540/robot2024/Robot.java

```diff
@@ -94,13 +105,15 @@ public void robotPeriodic() {
 
         // Update mechanism visualiser in sim
         if (Robot.isSimulation()) MechanismVisualiser.periodic();
+//        robotContainer.leds.setPattern(Leds.Zone.ZONE1, SimpleLedPattern.alternating(Color.kBlueViolet, Color.kGreen));
     }
 
     /**
      * This function is called once when the robot is disabled.
      */
     @Override
     public void disabledInit() {
+        robotContainer.leds.setPattern(Leds.Zone.ELEVATOR_BACK, new LedPatternFlame());
```

Comment:
What if we used a more chill led pattern for disabled

Reply:
no.